iothub/service/src/RetryPolicies/ExponentialBackoffRetryPolicy.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Represents a retry policy that performs a specified number of retries, using a exponential back-off scheme, with option jitter,
+    /// to determine the interval between retries.
+    /// </summary>
+    /// <remarks>
+    /// Jitter can change the delay from 95% to 105% of the calculated time.
+    /// </remarks>
+    public class ExponentialBackoffRetryPolicy : RetryPolicyBase
```

Comment:
Same thing here and all the other copied types; we'd want to disambiguate by name, not just by namespace.